lib/ui/src/components/sidebar/Heading.stories.tsx

```diff
@@ -77,6 +79,7 @@ export const LinkAndText: Story = () => {
           title: 'My title',
           url: 'https://example.com',
           image: null,
+          target: '_blank',
```

Comment:
We don't want to change our stories from default behavior, so these changes (anything that isn't `target: undefined`) should be removed/updated please.

Reply:
Removed unnecessary changes